docs/libraries/workflow/GLOSSARY.md

```diff
@@ -1,44 +1,46 @@
-TODO
-
 # Glossary
 
 ## Worker
 
-A process that's running workflows.
+A process that queries for pending workflows with a specific filter. Filter is based on which workflows are registered in the given worker's registry.
+
+## Registry
 
-There are usually multiple workers running at the same time.
+A collection of registered workflows.
```

Comment:
This makes it sound like I'd define a registry with several workflows
somewhere, then be able to re-use it in code to put several complex steps into a
single call.

It also sounds like it would be helpful to have a way to easily navigate all
existing registries in a similar way that I might go look for existing ops that do the
thing I want.

Reply:
> then be able to re-use it in code to put several complex steps into a
single call

What do you mean by this?

Registries exist within code and do not need to be searched for anything by the developer. Its a way to tell the worker what workflows you want it to run. You define a registry in the /worker of each pkg and then in the monolith worker you import and merge them together when starting the workflow worker. They dont deal with ops at all